addons/l10n_es/data/account_data.xml

```diff
@@ -69,5 +69,8 @@
         <record id="tax_group_iva_5" model="account.tax.group">
             <field name="name">IVA 5%</field>
         </record>
+        <record id="tax_group_iva_0" model="account.tax.group">
```

Comment:
Already present in https://github.com/odoo/odoo/pull/108846/files#diff-be65fc06bf41c02157a65fa726a0c8e3cbc7c477eafa7fc9dc0fb87ae5d76d9fR6